src/jit/macro.expr

```diff
@@ -0,0 +1,62 @@
+# -*-whitespace-*-
+
+(macro: ^getf (,object ,type ,field)
+   (load (addr ,object (&offsetof ,type ,field)) (&SIZEOF_MEMBER ,type ,field)))
+(macro: ^setf (,object ,type ,field ,value)
+    (store (addr ,object (&offsetof ,type ,field)) ,value (&SIZEOF_MEMBER ,type ,field)))
+(macro: ^cu_string (,a) (idx  (^getf (cu) MVMCompUnit body.strings) ,a ptr_sz))
```

Comment:
Hm, but strings may be lazily loaded, or do we guard against that case somewhere else?

Reply:
Nope, good point, this is an old legacy and should be fixed